pkgs/servers/teleport/default.nix

```diff
@@ -46,6 +45,10 @@ buildGoPackage rec {
 
   buildTargets = [ "full" ];
 
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
+
   postInstall = ''
     install -Dm755 -t $client/bin $out/bin/tsh
```

Comment:
OOI is it deliberate that only the $out variant gets xdg-utils on its path?

Reply:
Genuinely do not know, but added the wrapper just in case.